pom.xml

```diff
@@ -2165,6 +2166,9 @@
               <include>org.eclipse.jetty:jetty-security</include>
               <include>org.eclipse.jetty:jetty-util</include>
               <include>org.eclipse.jetty:jetty-server</include>
+              <include>com.sun.jersey:jersey-core</include>
+              <include>com.sun.jersey:jersey-json</include>
+              <include>com.sun.jersey:jersey-server</include>
```

Comment:
javax.ws.rs is actually inside the Jersey artifacts.

The shading didn't work as I expected regardless if I added the javax.rs.ws stuff to here as well though.